src/App.js

```diff
@@ -24,12 +24,15 @@ function App() {
   const setToken = token => {
     setCookie("sessionToken", token, { path: "/" });
   };
+  const deleteToken = () => {
+    removeCookie("sessionToken", { path: "/" });
```

Comment:
you might want to move "sessionToken" and "path" into a constant instead of duplicating here?

Reply:
like this:
`const name = "sessionToken";
  const path = "/";
  const setToken = token => {
    setCookie(name, token, { path: path });
  };`
or this:
` const sessionToken = { name: "sessionToken", path: "/" };
  const setToken = token => {
    setCookie(sessionToken.name, token, { path: sessionToken.path });
  };`?